ChaRo-iOS/ChaRo-iOS/Source/Views/VCs/CreatePostScene/MapScene/AddressMainVC.swift

```diff
@@ -142,6 +150,35 @@ class AddressMainVC: UIViewController {
     }
 }
 
+// MARK: - Guide Animation
+extension AddressMainVC{
+    func setupGuideAnimationView(){
+        if !isFirstOpen {return}
```

Comment:
P2; early exit 하는 부분은 guard 문으로 하는게 가독성측면에서 더 좋지 않을까요?
```suggestion
        guard isFirstOpen else { return }
```

Reply:
오호 이게 더 깔끔하네요! 반영하겠습니다ㅎㅎ

Reply:
반영했습니다 :) https://github.com/TeamChaRo/ChaRo-iOS/pull/192/commits/911da0e3610af78cdef7bfeb97cd5fa38707173a